build/push-aws.sh

```diff
@@ -14,30 +14,16 @@
 
 set -eux
 
+source $(dirname $0)/build-common.sh
+
 BUCKET_NAME="cockroach"
 LATEST_SUFFIX=".LATEST"
 REPO_NAME="cockroach"
 SHA="${CIRCLE_SHA1-$(git rev-parse HEAD)}"
 
-# push_one_binary takes the path to the binary inside the repo.
-# eg: push_one_binary sql/sql.test
-# The file will be pushed to: s3://BUCKET_NAME/REPO_NAME/sql.test.SHA
-# The S3 basename will be stored in s3://BUCKET_NAME/REPO_NAME/sql.test.LATEST
-function push_one_binary {
-  rel_path=$1
-  binary_name=$(basename $1)
-
-  cd $(dirname $0)/..
-  time aws s3 cp ${rel_path} s3://${BUCKET_NAME}/${REPO_NAME}/${binary_name}.${SHA}
-
-  # Upload LATEST file.
-  tmpfile=$(mktemp /tmp/cockroach-push.XXXXXX)
-  echo ${SHA} > ${tmpfile}
-  time aws s3 cp ${tmpfile} s3://${BUCKET_NAME}/${REPO_NAME}/${binary_name}${LATEST_SUFFIX}
-  rm -f ${tmpfile}
-}
+OSARCH="linux-amd64"
 
-push_one_binary cockroach
-push_one_binary sql/sql.test
-push_one_binary acceptance/acceptance.test
-push_one_binary static-tests.tar.gz
+push_one_binary cockroach cockroach-${OSARCH}
```

Comment:
you're passing two arguments to a 1-arg function.

Also, please hold before merging this, or it will break all the nightly stuff.